<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pB10">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB50">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB100">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB500">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB10000">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10000!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBMAX">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>MAX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBreset">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBcoffee">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBmilk">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtea">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>tea(150)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
